### ISSUE TITLE
Argument `--force-unique` does not exist

### ISSUE THREAD

**Issue by syphernl** [iamjackg/md2cf#14]
When running this for a stack of our MD files we got an error stating:

```
Some documents have the same title. Update them or use --force-unique:
```

However, the `--force-unique` argument does not exist

**Comment by iamjackg** [comment]
You're right. Somebody else also reported this but I completely forgot to fix it in the last release! I'll fix it now.

**Comment by iamjackg** [comment]
Actually I think I realized why I ended up never actually implementing this in the first place: it's not really possible without breaking other functionality.

Implementing this would require having a way to make each title unique. This can be done in a number of ways (hash of the document contents, name of the parent page, absolute path of the markdown file), but the main issue is that all of these can change between runs for the same file. This means that there is no way to make sure that whatever additional identifier we add ends up being the same every time. 
 
This can lead to pages being accidentally overwritten, or modified pages being left behind in multiple copies. I'm leaning towards not implementing this at all.

I'm curious: what's your use case for this? One case I can think of is multiple `README.md` files scattered around a repo, but I'd like to know why you need this.

**Comment by syphernl** [comment]
> I'm curious: what's your use case for this? One case I can think of is multiple `README.md` files scattered around a repo, but I'd like to know why you need this.

We're currently in the process of migrating a different wiki (which generates Markdown files in structured folders in their export) to Confluence. However, sometimes there are "category" pages (e.g. `meetings`) which may exist in multiple folders.

**Comment by iamjackg** [comment]
That makes a lot of sense. Unfortunately I don't see a good general solution to this. What I can do is have the tool list all the files with conflicting titles so you can apply your own domain-specific transform.
